src/scss/App.scss

```diff
@@ -0,0 +1,184 @@
+$background-color-purple: #D0073E;
+$background-color-grey: #EDEDED;
+$text-dark-grey: #919191;
+$border-shadow: #D4D3D3;
+$border-dark-grey: #919191;
+$grey-three: #919191;
+
+@font-face {
+  src: url('../assets/fonts/Abadi-MT-Condensed-Extra-Bold.ttf');
+  font-family: Abadi;
+}
+
+@font-face {
+  src: url('../assets/fonts/AvenirNextLTPro-Cn.otf');
+  font-family: Avenir;
+}
+
+@font-face {
+  src: url('../assets/fonts/nba-spurs.ttf');
+  font-family: spurs;
+}
+
+body {
+  background-color: $background-color-main;
+  margin: 0;
+}
+
+.app {
+  text-align: center;
+}
+
+.active {
+  color: $background-color-purple !important;
+  border-bottom: 4px solid $background-color-purple;
+}
+
+.players {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-evenly;
+  list-style: none;
+  font-family: Abadi;
+  height: 45px;
+  box-shadow: 1px -1px 2px #D4D3D3;
+
+  .team {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    position: relative;
+    background: $background-color-purple;
+
+    .logo-container {
+      display: flex;
+      background-color: white;
+      border-radius: 24px;
+      box-shadow: 1px 1px 5px $border-dark-grey;
+      margin: 0 15px;
+    }
+
+    button {
+      background: none;
+      color: inherit;
+      border: none;
+      padding: 0;
+      cursor: pointer;
+      outline: inherit;
+      color: white;
+      text-transform: uppercase;
+      font-size: 22px;
+      font-family: spurs;
+      margin: 0 15px 0 0;
+
+      i {
+        display: inline-block;
+        vertical-align: middle;
+        margin-left: 10px;
+      }
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      left: 100%;
+      z-index: 99;
+      border-top: 45px solid #D0073E;
+      border-right: 20px solid transparent;
+    }
+  }
+
+  .links {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex: 1.5;
+    background-color: white;
+    font-family: Avenir;
+    text-transform: uppercase;
+    font-size: 16px;
+  
+    .link-container {
+      display: inline-block;
+      height: 100%;
+      margin: 0 25px;
+    }
+  
+    a {
+      color: black;
+      display: inline-block;
+      height: calc(100% - 19px);    
+      padding-top: 15px;
+      text-decoration: none;
+  
+      &:hover {
+        height: calc(100% - 19px);
+        color: $grey-three;
+      }
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      left: 100%;
+      z-index: 99;
+      border-top: 45px solid white;
+      border-right: 20px solid transparent;
+    }
+  }
+
+  .user {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    background: $background-color-grey;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 14px;
+    color: $text-dark-grey;
+    padding-left: 15px;
+  
+    div {
```

Comment:
The spacing for the user menu is off from the design and a little incorrect:

<img width="335" alt="Screen Shot 2020-02-13 at 10 34 08 AM" src="https://user-images.githubusercontent.com/623755/74450666-7378d080-4e4c-11ea-8eb6-10f62c292bf5.png">

Should be cleaned up, first off change this `div {` to be:

```
> * {
      display: flex;
      margin-left: 20px;
}

```

Next remove the `margin: 0 15px 0 0` on the Button below.

And align items accordingly.

Reply:
I've made the changes but we should have a chat about taking measurements in Lunacy. There are times where I see the distance in pixels, but when I implement that number in margin and padding it doesn't look the same in my browser. So I'm left with having to choose between the pixel number I see in Lunacy, and eyeballing it with a different number.

Reply:
In this case, just try to make it look as close as possible. Sketch/Lunacy isn't always reliable for every design/dev case (also it might also be a design flaw).

Eitherway, it's not a big deal, we can take this convo to slack, np

Reply:
This is Still not changed, let's do this:

```
> * {
      display: flex;
      padding: 0 20px;
      height: 100%;
}
```